test/unit/event/send-ack-event.test.js

```diff
@@ -17,26 +17,26 @@ jest.mock('ffc-pay-event-publisher', () => {
   }
 })
 jest.mock('../../../app/processing/get-payment-request-by-invoice-frn')
-const getPaymentSchemeByInvoiceAndFrn = require('../../../app/processing/get-payment-request-by-invoice-frn')
+const { getPaymentRequestByInvoiceAndFrn } = require('../../../app/processing/get-payment-request-by-invoice-frn')
 jest.mock('../../../app/config')
-const config = require('../../../app/config')
+const { processingConfig, messageConfig } = require('../../../app/config')
 const { PAYMENT_ACKNOWLEDGED } = require('../../../app/constants/events')
 const { SOURCE } = require('../../../app/constants/source')
-const sendAckEvent = require('../../../app/event/send-ack-event')
+const { sendAckEvent } = require('../../../app/event/send-ack-event')
 
 let paymentRequest
 let acknowledgement
 
 beforeEach(() => {
-  paymentRequest = JSON.parse(JSON.stringify(require('../../mocks/payment-request')))
+  paymentRequest = JSON.parse(JSON.stringify(require('../../mocks/payment-requests/payment-request')))
   acknowledgement = JSON.parse(JSON.stringify(require('../../mocks/acknowledgement')))
 
-  getPaymentSchemeByInvoiceAndFrn.mockResolvedValue(paymentRequest)
+  getPaymentRequestByInvoiceAndFrn.mockResolvedValue(paymentRequest)
```

Comment:
Should the .mockResolvedValue be above the stringifys?

Reply:
It can't be as line 36 is dependent on 33